_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -194,7 +173,6 @@ class HealthCareApp extends React.Component {
   publishStateChange(propertyPath, update) {
     const newApplicationData = Object.assign({}, this.state.applicationData);
     _.set(newApplicationData, propertyPath, update);
-    window.localStorage['health-app-data'] = JSON.stringify(newApplicationData);
```

Comment:
Yeah...I had added that cause I was driving myself batty in the original code inputting test data. That's not an issue anymore since we have good unittests. Now it just causes problems so might as well delete.